orbs/phraseapp/README.md

```diff
@@ -0,0 +1,87 @@
+# Phraseapp Orb
+
+## Proposal
+This orb was created to make the merge of the changes made in the app.phrase with the GitHub project repository simple.
+
+## Configuration Needed
+1) Add [pharaseapp.yml](.phraseapp.sample.yml) to the root of your project.
+It will be used to know what locales we need to fetch from the app.phrase, the path to pull the changes and the format of the configuration file.
+```
+phraseapp:
+  available_locales:
+    en
+    fr
+    de
+  push:
+    sources:
+      - file: ./config/locales/<locale_name>.yml
+        params:
+          file_format: yml
+  pull:
+    targets:
+      - file: ./config/locales/<locale_name>.yml
+        params:
+          file_format: yml
+```
+
+
+2) Add jobtomate as a user to the project collaborators in the github repository. Instructions could be found [here](https://github.com/jobteaser/<repository>/settings/access).  
```

Comment:
`jobtomate` being an all-power bot user, it's better to avoid using it when possible. Would it be possible to use user  `circleci-jt` instead ?

Reply:
**remark**
From a "security" point of view (because this repo is public), not sure if it's a good thing to mention which user to use.